packages/client/helpers/build.ts

```diff
@@ -100,6 +101,20 @@ function writeDtsRexport(fileName: string) {
   fs.writeFileSync(path.join(runtimeDir, fileName), 'export * from "./index"\n')
 }
 
+function writeDefaultIndexFile() {
```

Comment:
wdyt about having a `script` folder in `src` with a ts file, and using the bundler to inject the right version? It might be less lines of code, but overall I am fine with this approach too.

Reply:
that sounds better, yep

Reply:
After trying this one thing I'm not super happy with is that the bundler can't tree-shake the json files, which leads the whole `package.json` contents of both `@prisma/client` and `@prisma/engines-version` to be included into the bundle. Together with a little bit of other generated boilerplate, this inflates the `default-index.js` to 9 Kb.

Do you think it's acceptable or should we rather not do it?

Reply:
I thought it could tree-shake things via imports like `import { version } from "./package.json"`, if that is not the case then drop it.

Reply:
Indeed! The problem was I used `clientVersion` from `runtime/utils`, and it used `require` rather than `import` (and, likewise, so does `@prisma/engines-version`). I pushed a commit now.

I also tried moving the types to the `.ts` file and generating the `.d.ts` but couldn't get it to work properly (the generated `.d.ts` file looked like `export * from "/Users/aqrln/prisma/prisma/packages/client/src/scripts/default-index.ts"`).